docs/reference/aggregations/reducer/movavg-aggregation.asciidoc

```diff
@@ -223,13 +223,108 @@ to see.  Small values emphasize long-term trends (such as a constant linear tren
+
+===== Additive Holt-Winters
+
+Additive seasonality is the default; it can also be specified by setting `"type": "add"`.  This variety is preferred
+when the seasonal affect is additive to your data. E.g. you could simply subtract the seasonal effect to "de-seasonalize"
+your data into a flat trend.
+
+The default value of `alpha`, `beta` and `gamma` is `0.5`, and the settings accept any float from 0-1 inclusive.
+The default value of `period` is `1`.
+
+[source,js]
+--------------------------------------------------
+{
+    "the_movavg":{
+        "moving_avg":{
+            "buckets_path": "the_sum",
+            "model" : "holt_winters",
+            "settings" : {
+                "type" : "add",
+                "alpha" : 0.5,
+                "beta" : 0.5,
+                "gamma" : 0.5,
+                "period" : 7
+            }
+        }
+}
+--------------------------------------------------
+
+
+[[holt_winters_add]]
+.Holt-Winters moving average with window of size 120, alpha = 0.5, beta = 0.7, gamma = 0.3, period = 30
+image::images/reducers_movavg/triple.png[]
+
+===== Multiplicative Holt-Winters
+
+Multiplicative is specified by setting `"type": "mult"`.  This variety is preferred when the seasonal affect is
+multiplied against your data. E.g. if the seasonal affect is x5 the data, rather than simply adding to it.
+
+The default value of `alpha`, `beta` and `gamma` is `0.5`, and the settings accept any float from 0-1 inclusive.
+The default value of `period` is `1`.
+
+[WARNING]
+======
+Multiplicative Holt-Winters works by dividing each data point by the seasonal value.  This is problematic if any of
+your data is zero, or if there are gaps in the data (since this results in a divid-by-zero).  To combat this, the
+`mult` Holt-Winters pads all values by a very small amount (1*10^-10^) so that all values are non-zero.  This affects
+the result, but only minimally.  If your data is non-zero, or you prefer to see `NaN` when zero's are encountered,
+you can disable this behavior with `pad: false`
+======
+
+[source,js]
+--------------------------------------------------
+{
+    "the_movavg":{
+        "moving_avg":{
+            "buckets_path": "the_sum",
+            "model" : "holt_winters",
+            "settings" : {
+                "type" : "mult",
+                "alpha" : 0.5,
+                "beta" : 0.5,
+                "gamma" : 0.5,
+                "period" : 7,
+                "pad" : true
+            }
+        }
+}
+--------------------------------------------------
+
+===== "Cold Start"
+
+Unfortunately, due to the nature of Holt-Winters, it requires two periods of data to "bootstrap" the algorithm.  This
+means that your `window` must always be *at least* twice the size of your period.  An exception will be thrown if it
+isn't.  It also means that Holt-Winters will not emit a value for the first `2 * period` buckets; the current algorithm
+does not backcast.
```

Comment:
Given we don't backcast, it might be better if the images associated with the Holt-Winters examples shows the data for the `2 * period` time when the Holt-Winters prediction isn't available to re-iterate the fact that it won't be emitted for the start of the data?